packages/cli/lib/nango.yaml.schema.v2.json

```
@@ -18,6 +18,29 @@
                             "_": "post-connection-scripts must be an array of strings."
                         }
                     },
+                    "on-events": {
+                        "type": "object",
```

```suggestion
                        "type": "object",
                        "additionalProperties": false,
```
Had a typo and could not figure out why I didn't got any error